modules/standard/Spawn.chpl

```diff
@@ -813,11 +824,99 @@ module Spawn {
     if err then ioerror(err, "in subprocess.close");
   }
 
-// Future work: support
-// send_signal
-// terminate
-// kill (send SIGKILL)
+  // Signals as required by POSIX-2013
+  enum signal {
+    SIGABRT = 1,
+    SIGALRM,
+    SIGBUS,
+    SIGCHLD,
+    SIGCONT,
+    SIGEMT,
+    SIGFPE,
+    SIGHUP,
+    SIGILL,
+    SIGINFO,
+    SIGINT,
+    SIGIO,
+    SIGKILL,
+    SIGPIPE,
+    SIGPOLL,
+    SIGPROF,
+    SIGQUIT,
+    SIGSEGV,
+    SIGSTOP,
+    SIGSYS,
+    SIGTERM,
+    SIGTRAP,
+    SIGTSTP,
+    SIGTTIN,
+    SIGTTOU,
+    SIGURG,
+    SIGUSR1,
+    SIGUSR2,
+    SIGVTALRM,
+    SIGWINCH,
+    SIGXCPU,
+    SIGXFSZ
+  };
+
+  private extern proc qio_send_signal(pid: int(64), qio_sig: int(64)): syserr;
+
+  /*
+
+    :arg error: optional argument to capture any error encountered
+                when .
```

Comment:
I know docs-commenting is mostly a TODO for this PR. Pointing out this sentence is only partly written, in case it is an oversight.